test/src/androidMain/kotlin/io/realm/util/PlatformUtils.kt

```diff
@@ -41,4 +42,36 @@ actual object PlatformUtils {
     }
 
     actual fun threadId(): ULong = Thread.currentThread().id.toULong()
+
+    @Suppress("ExplicitGarbageCollectionCall")
+    actual fun triggerGC() {
+        for (i in 1..2) {
+            allocGarbage(0)
+            SystemClock.sleep(100)
+            System.gc()
+            System.runFinalization()
+        }
+        SystemClock.sleep(5000) // 5 seconds to give the GC some time to process
```

Comment:
Is there any, even anecdotal, evidence this does anything useful? Sleeping for 5 seconds is pretty heavy.

Reply:
I just moved the code from https://github.com/realm/realm-kotlin/pull/300/files#diff-8243d5deaa5b3325464776989c0be50c922a20b96ac486be0e9fdf60f026c011L165 to `PlatformUtils` to enable using `triggerGC` in common tests. Only comment that I accidentally didn't copy is `Empiric approach to trigger GC`. I added it again, but maybe @nhachicha can elaborate a bit?